installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,228 @@
+scale, manage, and upgrade your workers.
+////
+
+include::modules/installation-cloudformation-worker.adoc[leveloffset=+2]
+//Do we need to remove the instructions to create separate worker nodes? It looks like the installation program might make the workers because we're creating a manifest for them.
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* You can view details about the CloudFormation stacks that you create by navigating to the link:https://console.aws.amazon.com/cloudformation/[AWS CloudFormation console].
+
+include::modules/installation-aws-user-infra-bootstrap.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../support/troubleshooting/troubleshooting-installations.html#monitoring-installation-progress_troubleshooting-installations[Monitoring installation progress] for details about monitoring the installation, bootstrap, and control plane logs as an {product-title} installation progresses.
+
+* See xref:../../support/troubleshooting/troubleshooting-installations.adoc#gathering-bootstrap-diagnostic-data_troubleshooting-installations[Gathering bootstrap node diagnostic data] for information about troubleshooting issues related to the bootstrap process.
+
+* You can view details about the running instances that are created by using the link:https://console.aws.amazon.com/ec2[AWS EC2 console].
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-operators-config.adoc[leveloffset=+1]
+
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
+
+You can configure registry storage for user-provisioned infrastructure in AWS to deploy {product-title} to hidden regions. See xref:../../registry/configuring_registry_storage/configuring-registry-storage-aws-user-infrastructure.adoc#configuring-registry-storage-aws-user-infrastructure[Configuring the registry for AWS user-provisioned infrastructure] for more information.
+
+include::modules/registry-configuring-storage-aws-user-infra.adoc[leveloffset=+3]
+
+include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+
+include::modules/installation-aws-user-infra-delete-bootstrap.adoc[leveloffset=+1]
+
+include::modules/installation-create-ingress-dns-records.adoc[leveloffset=+1]
+
+include::modules/installation-aws-user-infra-installation.adoc[leveloffset=+1]
+
+include::modules/logging-in-by-using-the-web-console.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../web_console/web-console.adoc#web-console[Accessing the web console] for more details about accessing and understanding the {product-title} web console.
+
+include::modules/cluster-telemetry.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
I think we can add a few resources here, something like:
- ["Opt in to Local Zones"](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#opt-in-local-zone)
- Local Zones available locations https://aws.amazon.com/about-aws/global-infrastructure/localzones/locations
- Local Zones features https://aws.amazon.com/about-aws/global-infrastructure/localzones/features

Reply:
I'm moving these links down to the `== Additional resources` section down a few lines because they apply to the whole document.